HTMLTestRunner.py

```diff
@@ -668,9 +668,10 @@ def _generate_report(self, result):
                 desc = desc,
                 count = np+ns+nf+ne,
                 Pass = np,
+                skip = ns,
                 fail = nf,
                 error = ne,
-                cid = 'c%s' % (cid+1),
```

Comment:
@howareyou The trailing comma is syntactically correct in Python. It is useful for adding lines later without raising a syntax error when someone forgets that a comma was actually needed in the preceding.

Other than your personal preference, is there some reason for removing it? If not, then it complicates the git history without adding value. I suggest removing it from the commit.